english/wakeup/wake.md

```diff
@@ -0,0 +1 @@
+i woke up in the middle of the night, i ask here what time is it, she looked at me and say there is no time here in this world just enjoy the eternity this how every thing begin.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.